odk1-src/central-install-digital-ocean.rst

```diff
@@ -0,0 +1,240 @@
+ - If you plan on setting up DKIM (see below), you will want to set the name of the server to the full domain name you intend to host your server.
+ - If you are technically savvy and understand what an SSH key is, there is a field here that you will want to fill in. If not, don't worry about it.
+
+Once you click on **Create**, you'll be taken back to the Droplet management page. It may think for a moment, and then your new server should appear. Next to it will be an IP address, which should look something like ``183.47.101.24``. This is where your server is publicly located on the Internet. Don't worry, nobody can do anything with it until you let them.
+
+Congratulations! With those steps, you have now created a new server which you can access over the Internet, and started it up. Next, we will get a web domain name address (like ``opendatakit.org``) to point at it.
+
+.. _central-install-digital-ocean-domain:
+
+Obtaining a Web Address (Domain Name)
+-------------------------------------
+
+Now is the time to set up a domain name. We will do so, and then configure it so that it sends users to the server you created in the previous step.
+
+You'll need to do this for two reasons: a memorable name (like ``google.com``) will be easier to remember and access than a pile of numbers, and you cannot obtain a security certificate without one. It is not currently possible to host ODK Central within a subdirectory on another domain (so, ``my-website.com/my-odk-server`` is not possible, but ``my-odk-server.com`` is allowed, as is ``my-odk-server.my-website.com``).
+
+If you already know how to do these sorts of things, feel free to ignore the following instructions and proceed on your own. You can rejoin us at the next section.
+
+For the rest of us, there are some options here:
+
+ - You can pay one of the many popular commercial domain registrars for a full domain name, like ``MyOdkCollectionServer.com``. Search for "domain registrar" to find one of these. These often cost as little as $3/year.
+ - You can use a free DNS service: we recommend `FreeDNS <https://freedns.afraid.org/>`_, which has run for a long time and has a good reputation. With it, you can obtain a free name, albeit with a fixed second half (like ``MyOdkCollectionServer.dynet.com``). If you choose this route, we recommend using one of the *less popular* names, as the heavily occupied names can run into trouble later on (in particular, obtaining a security certificate from Let's Encrypt).
+
+Whichever option you choose, once you obtain a domain name you'll want to look at `DigitalOcean's guide <https://www.digitalocean.com/community/tutorials/how-to-set-up-a-host-name-with-digitalocean>`_ on setting up domain names for your Droplet. In general, you'll point your domain name in DigitalOcean's direction at your registrar, then in DigitalOcean itself you'll want to create an A record that points to the IP address we found above.
```

Comment:
i pretty strongly dislike sending people off into the wilderness on their own without any kind of concrete link.. is there something so awful about using DO's DNS?

Reply:
On the Aggregate side, people were confused because this link made it seem like they *had* to use DO's DNS. Currently, your text reads like they have to move their domain's name servers to DO and then do the mapping. Chances are if they have a registrar, that registrar already has a DNS that they can use and that seems more straightforward.